### PR TITLE
Prepare 3.0.0

### PR DESCRIPTION
**Pull request by MattHJensen**
This prepares 3.0.0. 

Pending resolution of #2450. 

@Peter-Metz, could you look this over if you have an oportunity?

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -86,7 +86,7 @@ TAXCALC_JSON_FILES := $(shell ls -l ./taxcalc/*json | awk '{print $$9}')
 TESTS_JSON_FILES := $(shell ls -l ./taxcalc/tests/*json | awk '{print $$9}')
 PYLINT_FILES := $(shell grep -rl --include="*py" disable=locally-disabled .)
 PYLINT_OPTIONS = --disable=locally-disabled --score=no --jobs=4
-RECIPE_FILES := $(shell ls -l ./docs/cookbook/recipe*py | awk '{print $$9}')
```
[the remark this answers is not in the document]
Comment (MattHJensen) [reply Makefile:89]: I haven't used make cstest in the releasing process.

**Comment by MattHJensen** [comment]
I'll look at this JB failure later today. May have something to do with https://github.com/PSLmodels/Tax-Calculator/pull/2476 given that latest commit to this PR was rather innocuous.

**Comment by hdoupe** [comment]
@MattHJensen this looks like a flaky failure due to the anaconda service being unavailable temporarily:

Screenshot from 2020-08-24 08-06-41

I just restarted the tests.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (MattHJensen) [reply Makefile:89]: @Peter-Metz and I discussed on chat and he will be opening a separate issue on cstest.

**Pull request merged**